en-US/main.lang

```diff
@@ -869,3 +869,13 @@ Subscribe to monthly updates and keep current with Mozilla news, including the l
 
 ;Get Firefox tips, tricks, news and more
 Get Firefox tips, tricks, news and more
+
+
+# This  string is a potential navigation label on Technology page
+;Innovations
+Innovations
+
+
+# This  string is a potential navigation label on Technology page
```

Comment:
Let's fix this comment.

"This string will be used as a navigation label in home page, pointing to /technology"